src/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <main>
+      <div class="container-md">
+        <h1>Leaderboard</h1>
+        <div class="row mx-0">
+          <section id="recent" class="col-6">
+            <span class="h2">Recent scores</span>
+            <button type="button" class="btn btn-primary">Refresh</button>
+            <table class="table table-striped">
+              <tbody>
+                <tr>
+                  <td>Name: 100</td>
+                </tr>
+                <tr>
+                  <td>Name: 20</td>
+                </tr>
+                <tr>
+                  <td>Name: 50</td>
+                </tr>
+                <tr>
+                  <td>Name: 78</td>
+                </tr>
+                <tr>
+                  <td>Name: 125</td>
+                </tr>
+                <tr>
+                  <td>Name: 77</td>
+                </tr>
+                <tr>
+                  <td>Name: 42</td>
+                </tr>
+              </tbody>
```

Comment:
- [ ] It will be nice to dynamically render these list items. It will give you a chance to implement `arrow functions` and other ES6 features as it is a requirement for this project.

Reply:
Thanks! I implemented this suggestion.